src/components/Alert.tsx

```diff
@@ -15,7 +15,7 @@ const Alert = (props: {
 
   return (
     <div
-      className={`flex items-start space-x-2 text-left bg-warning-light rounded-2xl p-3 text-sm h-fit ${props.className}`}
+      className={`flex items-start space-x-2 text-left bg-warning-light rounded-2xl p-3 text-sm h-fit bg-alert-yellow ${props.className}`}
```

Comment:
I think we can remove `bg-warning-light` since we're using `bg-alert-yellow` as default